### ISSUE TITLE
DietPi-Software | Enable Chromium on ARMv6 (RPi1/Zero)

### ISSUE THREAD

**Issue by MichaIng** [MichaIng/DietPi#2860]
@Fourdee 
Do you know why we disabled it? Actually since all ARMv6 are RPi and we simply install the packages from Raspberry Pi firmware repo, this should work.

Ref:
- https://dietpi.com/phpbb/viewtopic.php?f=9&t=5972
- https://dietpi.com/phpbb/viewtopic.php?f=11&t=5675

**Comment by symbios24** [comment]
chromium is very slow on rpi zero

**Comment by MichaIng** [comment]
@symbios24 
That would be a good reason actually. Would apply for all ARMv6 RPis. I just wonder a bid since Raspbian comes with Chromium as default browser for a while.

Some time back when I used desktop and browser on my RPi2, I also found Chromium and Firefox to be much less responsive then the default browser that time: `netsurf`
- But AFAIK since hardware acceleration and such have been applied they switched to Chromium. But no experience about speed.

However if it generally works, I think it's okay to leave choice at user if it's too slow or not, as long as it works fine otherwise.
